src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Persons/PersonsControllerTests.cs

```diff
@@ -17,9 +18,66 @@ public async Task CreateSavedFilter_AsViewer_ShouldSaveFilter()
                 "criteria",
                 true);
 
+            var savedFilters = await PersonsControllerTestsHelper.GetSavedFiltersInProject(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                null);
+
+            
+            var savedFilter = savedFilters.Find(sf => sf.Id == id);
+
             // Assert
             Assert.IsTrue(id > 0);
-            //todo: when get saved filters is complete we can get and assert
+            Assert.IsTrue(savedFilters.Count > 0);
+            Assert.IsNotNull(savedFilter);
+            Assert.AreEqual(savedFilter.Title, "test title");
+            Assert.AreEqual(savedFilter.Criteria, "criteria");
+        }
+
+        [TestMethod]
+        public async Task GetSavedFiltersInProject_AsViewer_ShouldGetFilters()
+        {
+            // Act
+            var id = await PersonsControllerTestsHelper.CreateSavedFilter(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                "test title2",
+                "criteria",
+                true);
+
+            var savedFilters = await PersonsControllerTestsHelper.GetSavedFiltersInProject(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                null);
+
+            var savedFilter = savedFilters.Find(sf => sf.Id == id);
+
+            // Assert
+            Assert.IsTrue(savedFilters.Count > 0);
```

Comment:
Should have more than one filter na test should assert on a a specific number of results to avoid hiding errors